fastlane/docs/Codesigning/XcodeProject.md

```diff
@@ -31,6 +31,18 @@ PROVISIONING_PROFILE_SPECIFIER = "[team_id]/[profile_name]";
 
 You could pass the profile specifier via environment variables too (See Xcode 7 approach below), however in most cases it's enough to define it statically in your project, per target, since usually neither your Team ID, nor the provisioning profile change.
 
+If you want to set it dynamically, you can use sigh or match to fill the environment variables:
```

Comment:
Cleaned it up a bit. I don't know how to describe the match behaviour. Match generates different environment variables for every build type, so I use an environment variable with a constant name between Xcode and match.

```
ENV['PROFILE_NAME'] = ENV['sigh_tools.fastlane.test_profile_name_app_store'] #app store lane
ENV['PROFILE_NAME'] = ENV['sigh_tools.fastlane.test_profile_name_adhoc'] #adhoc lane

Xcode config:
PROVISIONING_PROFILE_SPECIFIER = $(FASTLANE_TEAM_ID)/$(PROFILE_NAME)
```